experiment/measurer.py

```diff
@@ -387,6 +387,13 @@ def get_current_coverage(self) -> int:
 
     def generate_profdata(self, cycle: int):
         """Generate .profdata file from .profraw file."""
+        if not os.path.exists(self.profraw_file):
```

Comment:
So this will log the same error about there not being a profraw file twice? Once in generate_profdata and once in generate_summary? Maybe we should have a helper method that checks for these files and returns early if they don't exist, otherwise it calls both of these functions. This way we can log error once.

Reply:
Yes i will fix that in next CL, at first i am debugging other things. Basically all runner exits get fixed after this. Now, the next challenge is some coverage merge failures - see http://commondatastorage.googleapis.com/fuzzbench-reports/aarya-2020-07-26/index.html